src/content/api/cli.md

```diff
@@ -459,10 +459,10 @@ NODE_OPTIONS="--max-old-space-size=4096 -r /path/to/preload/file.js" webpack
 You can use the above environment variables inside your webpack configuration:
 
 ```javascript
-   module.exports = {
-     // ...
-     mode: process.env.WEBPACK_SERVE ? 'development' : 'production',
-   }
+module.exports = {
+  // ...
+  mode: process.env.WEBPACK_SERVE ? 'development' : 'production',
+}
```

Comment:
It is not `process.env.WEBPACK_SERVE`, it should be `module.exports = (env, args) => { return { mode: env.WEBPACK_SERVER ? 'development' : 'production' };  }`. it is not safe to change env variables

Reply:
fixed